### DISCUSSION TITLE
send response for particular task using python fast api

### DISCUSSION THREAD

**Discussion by santosh614696** [PrefectHQ/prefect#15986] [Q&A]
1. is it possible to send / push a response / message for particular task using python fast api, task as to wait until it get response from api
2. based on response we have trigger respective subflow, and close a particular subflow.

**Comment by desertaxle** [comment]
Hey @santosh614696! It's possible to send and receive inputs to running flows, but not tasks. Here's a link to the documentation for sending and receiving message to running flows: https://docs.prefect.io/3.0/develop/inputs.

For your second point, run a child flow based on the received input should be doable via pure Python. You can read the received input and use `if/elif/else` statements to choose which flow to run. Stopping a flow run depends on how it was started, so I'll need more information to suggest an approach.
